assignment_3/SVM_by_subham/README.md

```diff
@@ -0,0 +1,24 @@
+## Approach
+
+1. Generated the toy dataset using make_blobs
+	- Approach
+2. Converted the data into a dictionary data_dict with positive class as 1 and negative as -1.
+
+3. Made a class named SVM with 3 method.
+    -fit
+    -predict
+    -visualize
+
+4. In method fit, searched for the optimum w and optimum b in the range of -(maximum feature value) to +(maximum feature value) initialy and then decreased the range with 0.1 then with 0.01 and lastly with 0.001. If any w and b are the required w and b then checked the condition yi*(xi*w+b) >= 1 to be true.
+
+5. Visualised the data with the method visualize. 
```

Comment:
Here you can add the visualized result .png or .jpeg file of the matplotlib figure.
like this 
![image](https://user-images.githubusercontent.com/23621655/57805869-80607a80-777b-11e9-87fb-c84695a561e1.png)